cmd/crates/soroban-spec-typescript/src/lib.rs

```diff
@@ -293,35 +303,27 @@ pub fn entry_to_method_type(entry: &Entry) -> String {
 ",
             )
         }
-        Entry::ErrorEnum { doc, cases, .. } => {
-            let doc = doc_to_ts_doc(doc, None, 0);
-            let cases = cases
-                .iter()
-                .enumerate()
-                .map(|(i, c)| {
-                    if c.doc.is_empty() {
-                        format!(
-                            "{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            c.value,
-                            c.name
-                        )
-                    } else {
-                        format!(
-                            "{}{}  {}: {{message:\"{}\"}}",
-                            if i == 0 { "" } else { "\n" },
-                            doc_to_ts_doc(&c.doc, None, 1),
-                            c.value,
-                            c.name
-                        )
-                    }
-                })
-                .join(",\n");
-            format!("{doc}export const Errors = {{\n{cases}\n}}")
+        Entry::ErrorEnum { .. } => {
+            // type filtered out before entry_to_method_type is called
+            String::new()
         }
     }
 }
 
+fn error_cases_to_ts(cases: &[ErrorEnumCase]) -> Vec<String> {
+    cases
+        .iter()
+        .map(|c| {
+            format!(
+                "{}  {}: {{message:\"{}\"}}",
+                doc_to_ts_doc(&c.doc, None, 1),
+                c.value,
+                c.name
+            )
+        })
```

Comment:
```suggestion
        .map(|ErrorEnumCase { value, name, doc, .. } | {
            format!(
                "{value}  {name}: {{message:\"{}\"}}",
                doc_to_ts_doc(&doc, None, 1)
            )
        })
```

Reply:
Gotta love pattern matching on paramaters.

Reply:
Thanks! I moved this from another location but yeah, the pattern-matched version is 10x better.

Reply:
See the newest change for the correct version of this (you put some params in the wrong spots)